kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelService.kt

```diff
@@ -545,6 +545,10 @@ class TunnelService : VpnService() {
                                 running = false
                             }
 
+                            is Event.DnsRecordsChanged -> {
+                                // TODO: Save cache.
+                            }
```

Comment:
So good, got a Kotlin compile error because I changed the FFI code in Rust 😍 

We should adopt UniFFI for Apple soon.